docs/example/stable_diffusion.md

```diff
@@ -0,0 +1,18 @@
+This example provides a demo service for stable diffusion. You can develop this in the container environment by using [envd](https://github.com/tensorchord/envd): `envd up`.
+
+##### Server
+
+    envd build -t sd:serving
```

Comment:
This command should be run under `examples/stable_diffusion`?

Reply:
Yes, under the same folder with `build.envd` file. Let me add some comments to make it clear.

Reply:
Or if at root, can the command specify the file? Since the server starting command is at root level.

Reply:
Otherwise you can use `envd build -p examples/stable_diffusion -t sd:serving`